src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +72,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.